src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String SHORT_LOGIN = "short";
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String SHORT_PASSWORD = "short";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final Integer VALID_AGE = 20;
+    private static final Integer UNDERAGE = 17;
+    private static final String EXISTING_LOGIN = "existingLogin";
```

Comment:
It's good to see constants being used for test values. However, the names like `SHORT_LOGIN` and `SHORT_PASSWORD` do not indicate what is actually short about them. It would be better to reflect the actual length, for example, `SHORT_LOGIN_5_CHARS`. Also, ensure that these constants are indeed short according to the validation rules in `RegistrationServiceImpl`.